sirepo/html.py

```diff
@@ -12,7 +12,7 @@
 import sys
 
 
-_CALL_RE = re.compile(r'(\w+)\(([^\)]*)\);', flags=re.MULTILINE)
+_CALL_RE = re.compile(r'^\s+(\w+)\(([^\)]*)\);', flags=re.MULTILINE)
```

Comment:
This handles:
```html
<h4 class="text-center plan-name" style="background-image: url('/en/img/plans/sirepo-basic.jpg');">Sirepo<br>Basic</h4>
```
`url()` should not match.

Reply:
This seems OK to me - the call has to be the first item on the line.